packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -578,21 +578,33 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
         ? `${SERVERLESS_OBSERVABILITY_DOCS}create-custom-threshold-alert-rule`
         : `${OBSERVABILITY_DOCS}custom-threshold-alert.html`,
       monitorStatus: `${OBSERVABILITY_DOCS}monitor-status-alert.html`,
-      monitorUptime: `${OBSERVABILITY_DOCS}monitor-uptime.html`,
+      monitorUptime: isServerless
+        ? `${SERVERLESS_OBSERVABILITY_DOCS}monitor-synthetics`
+        : `${OBSERVABILITY_DOCS}monitor-uptime.html`,
       tlsCertificate: `${OBSERVABILITY_DOCS}tls-certificate-alert.html`,
       uptimeDurationAnomaly: `${OBSERVABILITY_DOCS}duration-anomaly-alert.html`,
-      monitorLogs: `${OBSERVABILITY_DOCS}monitor-logs.html`,
+      monitorLogs: isServerless
+        ? `${SERVERLESS_OBSERVABILITY_DOCS}discover-and-explore-logs`
```

Comment:
@mdbirnstiehl should confirm that this is the correct link (I'm not sure where the link appears in the UI).